test/e2e/webhook_test.go

```diff
@@ -714,7 +715,11 @@ func newManagedCluster(name string, accepted bool, externalURL string) *clusterv
 	return &clusterv1.ManagedCluster{
 		ObjectMeta: metav1.ObjectMeta{
 			Name: name,
+			Labels: map[string]string{
```

Comment:
Please cover cases labels nil as well as not nil.

Reply:
it could not be nil,
As it must be has `clusterSetLabels[clusterSetLabel] = defaultClusterSetName`